mitodl/mitxpro
# chore: run tests in parallel

### What are the relevant tickets?




None

### Description (What does it do?)

Run tests in parallel. This improves the test execution time by ~40%. The reduced time is ~13m. It was ~20+ minutes.

### How can this be tested?


- Checks should pass
- All tests should run
- Nothing should change apart from the improved execution time
- No impact on deployments
